lock/src/main/java/com/auth0/android/lock/Configuration.java

```diff
@@ -92,6 +95,25 @@ public Strategy getActiveDirectoryStrategy() {
         return activeDirectoryStrategy;
     }
 
+    @Nullable
+    public Strategy getDefaultPasswordlessStrategy() {
```

Comment:
Great, but I'll make it future-proof like this

```java
if (passwordlessStrategies.isEmtpy()) {
     return null;
}

if (passwordlessStrategies.size() == 1) {
    return passwordlessStrategies.get(0);
}

Strategy default = null;

for (Strategy s : passwordlessStrategies) {
    if (s.getName().equals(Strategies.Email.getName())) {
        default = s;
        break;
    }
}

return default || passwordlessStrategies.get(0);
```

If we have more than one we pick email or whatever is the first one in case there is no email connection